pkg/action/install.go

```diff
@@ -405,6 +407,14 @@ func writeToFile(outputDir string, name string, data string) error {
 	return nil
 }
 
+func createOrOpenFile(filename string, append bool) (*os.File, error) {
+	if append {
+		return os.OpenFile(filename, os.O_APPEND|os.O_WRONLY, 0600)
+	} else {
```

Comment:
You can drop the "else". Small 'linter' nit.

Reply:
@hickeyma Can you change this directly? Otherwise I'll do it in a week when I am back at my desk.

Reply:
Yiu can do it when you return @torstenwalter when you return. You have done all the hard work.

Reply:
@hickeyma Change is implemented.